### ISSUE TITLE
todo: seasonality procedure

### ISSUE THREAD

**Issue by epogrebnyak** [mini-kep/user-charts#5]
As a simple approximation of the task - in a dataset I have some highly seasonal time sereis like these ones: 
- <http://minikep-db.herokuapp.com/all/series/INVESTMENT_bln_rub/m>
- <http://minikep-db.herokuapp.com/all/series/TRANSPORT_FREIGHT_bln_tkm/m>

Can look up these in [macrodash](http://macrodash.herokuapp.com/)

As well as some time series where the seasonlaity is not apparent, but you also want to have it removed, as in [GDP](http://minikep-db.herokuapp.com/all/series/GDP_yoy/q)

Basically, I would like to store one or several seasonally adjusted versions for the indicator 
and deliver it in API with sa parameter 

<http://minikep-db.herokuapp.com/all/series/TRANSPORT_FREIGHT_bln_tkm/m/sa> (example, not working)

also sa can contain a verson like ```sa:x13```, `sa:something`

a problem for me is that my code is python and most seasonal adjustment is in R
[my own little snippet of R code](https://github.com/epogrebnyak/seasonality/blob/master/seasonality/example1.R)

**Comment by epogrebnyak** [comment]
Some fun with [R vs statmodels](https://github.com/statsmodels/statsmodels/issues/3549)

**Comment by epogrebnyak** [comment]
Will move to a separate folder:
- <https://github.com/mini-kep/user-charts/blob/master/seasonality.ipynb>
- <https://github.com/mini-kep/user-charts/blob/master/decomp.py>
